include/zcbor_common.h

```diff
@@ -386,6 +387,17 @@ bool zcbor_validate_string_fragments(struct zcbor_string_fragment *fragments,
 bool zcbor_splice_string_fragments(struct zcbor_string_fragment *fragments,
 		uint_fast32_t num_fragments, uint8_t *result, size_t *result_len);
 
+/** Compare two struct zcbor_string instances.
+ *
+ *  Return false if one one or both is NULL, and if length or contents don't match.
```

Comment:
Should we have the @param and @retval in these docs as we do for the other functions?

Reply:
Added